web/tests/EducationBenchmarking.Web.Integration.Tests/WhenViewingSchoolComparison.cs

```diff
@@ -127,8 +164,14 @@ private static void AssertPageLayout(IHtmlDocument page, School school)
         var comparisonComponent = page.GetElementById("compare-your-school");
         Assert.NotNull(comparisonComponent);
         
-        var toolsHeadingSection = page.Body.SelectSingleNode("//main/div[4]");
-        DocumentAssert.Heading2(toolsHeadingSection, "More tools");
-        //TODO: Get and assert more tools section on page
+        var toolsListSection = page.Body.SelectSingleNode("//main/div[4]");
```

Comment:
Probably something for the future, but it might be safer to select elements by some kind of static identifier rather than an index.

Reply:
👆🏼 This does seem to have been done in some places but not others